README.md

```diff
@@ -1 +1,169 @@
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Portofolio] <a name="about-project"></a>
+
+> My Portofolio 
+
+**[Portofolio]** is a program for my coding projects.
+
+## 🛠 Built With <a name="built-with"></a>
+
+ - HTML
+ - CSS
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Use_HTML]**
+- **[Use_CSS]**
+- **[Use_Linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> A viewer can easily see my projects.
+
+### Prerequisites
+
+In order to run this project you need: VS code
+
+
+### Setup
+
+Clone this repository to your desired folder: Clone the project or download the files to your local environment.
+
+
+### Install
+
+Install this project with: You have run the installation commands.
+
+
+### Usage
+
+To run the project, execute the following command: Use linters to check the errors in your html and style.
+
+
+### Run tests
+
+To run : Use test commands
+
+
+### Deployment
+
+You can deploy this project using: gcloud app deploy
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+- All collaborators
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Roland-Ntwali)
+- Twitter: [@twitterhandle](https://twitter.com/_Ntwali)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/roland-ntwali-11b16617b/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Projects]**
+- [ ] **[About_my_self]**
+- [ ] **[Contact_me]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Give a ⭐️ if you like this!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Have a great time!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please add a license file to the root of your folder, currently, the MIT link in the readme is showing a 404 error.